android/app/src/main/java/io/imagekit/imagekitdemo/Constants.kt

```diff
@@ -0,0 +1,15 @@
+package io.imagekit.imagekitdemo
+
+//Replace with ImageKit public key from ImageKit Dashboard (https://imagekit.io/dashboard/url-endpoints)
+const val IK_PUBLIC_KEY = "IK_PUBLIC_KEY"
+
+//Replace with ImageKit URL endpoint (or CNAME) from ImageKit Dashboard (https://imagekit.io/dashboard/url-endpoints)
+const val IMAGEKIT_ID = "IMAGEKIT_ID"
```

Comment:
This should be called `IK_URL_ENDPOINT`, not just IMAGEKIT_ID
the default value should be `https://ik.imagekit.io/IMAGEKIT_ID` as there may be certain users which have a url that can look like `https://imagekit.example.com`

Reply:
Done. Please check